scripts/install.sh

```diff
@@ -204,4 +204,17 @@ download_file_github \
     macro-circom \
     "${PREFIX}/bin"
 
+if $1 == "true" ; then
```

Comment:
I don't think passing a boolean argument to the script is the cleanest solution - when you see `./install.sh true` written in some other script, it doesn't really give you an idea what's the purpose of `true`.

I would rather have a CLI argument like `--enable-redis` or so. Here is an example how you can parse arguments like that in shell: https://github.com/Lightprotocol/install/blob/main/light-protocol-install.sh#L56-L83

Reply:
Good idea, will add the flag —enable-redis

Reply:
done